skxray/tests/test_roi.py

```diff
@@ -193,21 +169,47 @@ def _helper_check(pixel_list, inds, num_pix, q_ring_val, calib_center,
     data = ty.reshape(img_dim[0], img_dim[1])
 
     # get the grid values from the center
-    grid_values = core.pixel_to_radius(img_dim, calib_center)
+    grid_values = core.radial_grid(img_dim, center)
 
     # get the indices into a grid
     zero_grid = np.zeros((img_dim[0], img_dim[1]))
     for r in range(num_qs):
-        vl = (q_ring_val[r][0] <= grid_values) & (grid_values
-                                                  < q_ring_val[r][1])
+        vl = (edges[r][0] <= grid_values) & (grid_values
+                                                  < edges[r][1])
         zero_grid[vl] = r + 1
 
     # check the num_pixels
     num_pixels = []
     for r in range(num_qs):
         num_pixels.append(int((np.histogramdd(np.ravel(grid_values), bins=1,
-                                              range=[[q_ring_val[r][0],
-                                                      (q_ring_val[r][1]
+                                              range=[[edges[r][0],
+                                                      (edges[r][1]
                                                        - 0.000001)]]))[0][0]))
-    assert_array_equal(zero_grid, data)
     assert_array_equal(num_pix, num_pixels)
+
+
+def test_segmented_rings():
+    center = (20., 25.)
+    img_dim = (100, 80)
+    first_q = 5.
+    delta_q = 5.
+    num_rings = 4  # number of Q rings
+    slicing = 4
+
+    edges = roi.ring_edges(first_q, width=delta_q, spacing=4,
+                           num_rings=num_rings)
+    print("edges", edges)
+
+    label_array = roi.segmented_rings(edges, slicing, center,
+                                      img_dim, offset_angle=0)
+    print("label_array for segmented_rings", label_array)
+
+    label_mask, pixel_list = corr.extract_label_indices(label_array)
+    # number of pixels per ROI
+    num_pixels = np.bincount(label_mask, minlength=(np.max(label_mask)+1))
+    num_pixels = num_pixels
```

Comment:
To add more tests, If you want you can take that out, and we can merge this PR. I will work on adding more tests to these functions